File: 2016-reduction-operations-to-make-the-array-elements-equal/reduction-operations-to-make-the-array-elements-equal.py
class Solution:
    def reductionOperations(self, nums: List[int]) -> int:
        steps=0
        nums=sorted(nums)
        ele=nums[0]
        for i in range(len(nums)):
            if  ele<nums[i]:
                steps+=(len(nums)-i)
                ele=nums[i]
        return steps

File: 2016-reduction-operations-to-make-the-array-elements-equal/reduction-operations-to-make-the-array-elements-equal.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var reductionOperations = function(nums) {
     let steps = 0;
        nums.sort((a, b) => a - b); // Sort in ascending order
        let ele = nums[0];
        for (let i = 0; i < nums.length; i++) {
            if (ele < nums[i]) {
                steps += nums.length - i;
                ele = nums[i];
            }
        }
        return steps;
};
